client/src/components/NavBar.jsx

```diff
@@ -32,6 +32,9 @@ export default function NavBar({ loggedInUser, setLoggedInUser }) {
                   <NavItem>
                     <NavLink tag={RRNavLink} to="/userprofiles">
                       User Profiles
+                  </NavLink>
```

Comment:
You put the `NavLink` for the Categories List view within the `NavItem` for the User Profiles List view. Place the Categories List view `NavLink` within its own `NavItem` inside of the `Nav` element